pilot/pkg/serviceregistry/kube/conversion.go

```diff
@@ -153,6 +156,15 @@ func ConvertService(svc coreV1.Service, domainSuffix string, clusterID cluster.I
 	return istioService
 }
 
+func valueToStringGenerated(v interface{}) string {
+	rv := reflect.ValueOf(v)
```

Comment:
Why do we need reflection just to read a string?

Reply:
> Why do we need reflection just to read a string?

oh, this unnecessary, i will remove it